[2022-03-17 18:15:09,347] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: udac_example_dag.task_group_Dimension_table.load_time_dim_table scheduled__2022-03-17T09:00:00+00:00 [queued]>
[2022-03-17 18:15:09,367] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: udac_example_dag.task_group_Dimension_table.load_time_dim_table scheduled__2022-03-17T09:00:00+00:00 [queued]>
[2022-03-17 18:15:09,367] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 18:15:09,368] {taskinstance.py:1244} INFO - Starting attempt 1 of 3
[2022-03-17 18:15:09,368] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 18:15:09,384] {taskinstance.py:1264} INFO - Executing <Task(LoadDimensionOperator): task_group_Dimension_table.load_time_dim_table> on 2022-03-17 09:00:00+00:00
[2022-03-17 18:15:09,387] {standard_task_runner.py:52} INFO - Started process 23886 to run task
[2022-03-17 18:15:09,397] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'udac_example_dag', 'task_group_Dimension_table.load_time_dim_table', 'scheduled__2022-03-17T09:00:00+00:00', '--job-id', '142', '--raw', '--subdir', 'DAGS_FOLDER/udac_example_dag.py', '--cfg-path', '/var/folders/lh/ngst1lt51m9_gyz7qrg2f3nh0000gn/T/tmpui6gf2f6', '--error-file', '/var/folders/lh/ngst1lt51m9_gyz7qrg2f3nh0000gn/T/tmpn5qykvsd']
[2022-03-17 18:15:09,400] {standard_task_runner.py:77} INFO - Job 142: Subtask task_group_Dimension_table.load_time_dim_table
[2022-03-17 18:15:09,503] {logging_mixin.py:109} INFO - Running <TaskInstance: udac_example_dag.task_group_Dimension_table.load_time_dim_table scheduled__2022-03-17T09:00:00+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2022-03-17 18:15:09,777] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=udacity
AIRFLOW_CTX_DAG_ID=udac_example_dag
AIRFLOW_CTX_TASK_ID=task_group_Dimension_table.load_time_dim_table
AIRFLOW_CTX_EXECUTION_DATE=2022-03-17T09:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-17T09:00:00+00:00
[2022-03-17 18:15:09,780] {load_dimension.py:83} INFO - Starting to truncate time
[2022-03-17 18:15:09,793] {base.py:70} INFO - Using connection to: id: redshift. Host: redshift-cluster-1.cvmsgkpkbsh2.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: ***, extra: {}
[2022-03-17 18:15:11,124] {dbapi.py:225} INFO - Running statement: 
        TRUNCATE TABLE "public"."time"
        , parameters: None
[2022-03-17 18:15:11,832] {load_dimension.py:93} INFO - Starting to insert time
[2022-03-17 18:15:11,835] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-03-17 18:15:11,836] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'next_ds' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds }}' instead.
[2022-03-17 18:15:11,836] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2022-03-17 18:15:11,837] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2022-03-17 18:15:11,838] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_ds' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:15:11,839] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:15:11,839] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_execution_date' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:15:11,840] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'prev_execution_date_success' from the template is deprecated and will be removed in a future version. Please use 'prev_data_interval_start_success' instead.
[2022-03-17 18:15:11,841] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:15:11,842] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:15:11,843] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'yesterday_ds' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:15:11,844] {logging_mixin.py:109} WARNING - /usr/local/anaconda3/envs/data_pipeline/lib/python3.8/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'yesterday_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-03-17 18:15:11,858] {base.py:70} INFO - Using connection to: id: redshift. Host: redshift-cluster-1.cvmsgkpkbsh2.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: ***, extra: {}
[2022-03-17 18:15:13,226] {dbapi.py:225} INFO - Running statement: 
        INSERT INTO "public"."time" (start_time, hour, day, week, month , year, weekday)
 
        SELECT start_time, extract(hour from start_time), extract(day from start_time), extract(week from start_time),
               extract(month from start_time), extract(year from start_time), extract(dayofweek from start_time)
        FROM songplays
    
        , parameters: None
[2022-03-17 18:15:14,125] {dbapi.py:233} INFO - Rows affected: 6820
[2022-03-17 18:15:14,540] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=udac_example_dag, task_id=task_group_Dimension_table.load_time_dim_table, execution_date=20220317T090000, start_date=20220317T101509, end_date=20220317T101514
[2022-03-17 18:15:14,575] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-17 18:15:14,638] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
